autogen.sh

```diff
@@ -5,7 +5,7 @@
 # macros are not in default aclocal search path.
 #
 echo "Running libtoolize --automake --copy ... "
-libtoolize --automake --copy || exit
+libtoolize --force --automake --copy || exit
```

Comment:
did these libtoolize changes accidentally get put in?

Reply:
Yes, must have accidentally done `git commit -a`. I'll fix. Thanks!

Reply:
This should be fixed now